diff -aurN raul-0.5.0.orig/raul/List.hpp raul-0.5.0/raul/List.hpp
--- raul-0.5.0.orig/raul/List.hpp	2008-07-12 13:58:08.000000000 +0200
+++ raul-0.5.0/raul/List.hpp	2008-07-12 14:25:32.000000000 +0200
@@ -128,7 +128,7 @@
 		inline bool      operator==(const const_iterator& iter) const;
 	
 		friend class List<T>;
-		friend class List<T>::const_iterator;
+		//friend class List<T>::const_iterator;
 
 	private:
 		const List<T>*          _list;
